Mathlib/Data/Set/Intervals/Basic.lean

```diff
@@ -38,7 +38,7 @@ namespace Set
 
 section Preorder
 
-variable [Preorder α] {a a₁ a₂ b b₁ b₂ c x : α}
+variable [Preorder α] [Preorder β] {f : α → β} {a a₁ a₂ b b₁ b₂ c x : α}
```

Comment:
This change looks fishy to me

Reply:
Indeed, this had to do with the fact that Scott concurrently added the lemmas I was going to port here. Fixed now.